ui/components/data-sources/AddDataSourceDialog.js

```diff
@@ -13,7 +14,7 @@ const INITIAL_STATE = {
     },
     postgresOptions: {
         url: "",
-        port: 5432,
+        port: serverConfig.postgresConfig.port,
```

Comment:
We can't use `serverConfig.postgresConfig.port` here because those are two different databases. The one configured in `server/config.js` is the admin database while the one in the `AddDataSourceDialog` is a data source of type Postgres that can point to any database the user wants. The default value of 5432 is there because it's the default Postgres port (but the user can put in whatever they want).

Reply:
Yes, this one can be changed.